Source/swiftlint/Extensions/Configuration+CommandLine.swift

```diff
@@ -45,14 +45,15 @@ private func scriptInputFiles() -> Result<[File], CommandantError<()>> {
 }
 
 #if os(Linux)
-private func autoreleasepool(block: () -> Void) { block() }
+private func autoreleasepool<V>(block: () -> V) -> V { return block() }
```

Comment:
Nit: `T` is the standard generic type parameter name, not `V`. Especially since now this no longer necessarily means "Void".